### PR TITLE
Project 6. Apply for a review

### PR DESCRIPTION
**Pull request by GzimAsani**
This project was done with HML and CSS

Milestones:

 We started by creating a navbar for the smallest device I phone 5
 We created several sections which contain the necessary articles 
 We used display: none for the addition stuff that would appear in bigger screans
 We used display: none for navbar, footer and one section in the main.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,806 @@
...
+        </div>
+      </div>
+      <nav class='second-nav-mobile xxl'>
+        <p>
+          Join TNW2020, our 15th annual flagship event. Online . Get tickets ➔
+        </p>
+      </nav>
+    </div>
+    <section class="first-article">
+      <div class='first-article-cards-small article-card-ipad-flex'>
```
Comment (tirthajyoti-ghosh) [inline index.html:118]: The height of this section should be more to accommodate the headline. Right now the headline portion sticks out of its container.

image

Ideally, the height based on `vh` unit.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,806 @@
...
+              1 hour ago
+              <i class="fas fa-share-alt font-small-fa"></i>
+              6
+            </small>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="article-section article-section-ipad">
```
Comment (tirthajyoti-ghosh) [inline index.html:188]: Decrease the width and set a `max-width` for larger screens for this and all the sections below this - `latest funding`, `apps, gear, tech, ...`, and `latest deals` sections.

| Source Page	| Clone Page |
|:---:|:---:|
|Annotation 2020-09-09 221107|Annotation 2020-09-09 221020|

Comment (tirthajyoti-ghosh) [inline index.html:188]: There should be four boxes in each row in larger screens.

| Source Page	| Clone Page |
|:---:|:---:|
|image|image|

File: index.html [hunk 3]
```diff
@@ -0,0 +1,806 @@
...
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/11.jpeg" alt="article">
+          <div>
+            <h2>What are geofence warrants?</h2>
+            <small class="font-small-2x">Satoshi Nakaboto • 13 minutes ago</small>
+          </div>
+        </article>
+      </div>
+    </section>
+    <section class=" article-section-short article-section-ipad">
```
Comment (tirthajyoti-ghosh) [inline index.html:251]: The marked portion is sticking out of its container. Please fix this 👇 .

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,806 @@
...
+          </div>
+          <div class="funding-section-article-card-footer2">
+            <small>Mercuryo.io closes €2.5M Seed Funding</small>
+            <br>
+            <small class="font-small-3x">on FinSMEs</small>
+          </div>
+        </div>
+      </section>
+    </section>
+    <section class='dgrid2 article-section-short'>
```
Comment (tirthajyoti-ghosh) [inline index.html:339]: This section should have 3 boxes in each row in large screens.

| Source Page	| Clone Page |
|:---:|:---:|
|image|image|

File: index.html [hunk 5]
```diff
@@ -0,0 +1,806 @@
...
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+        </ul>
+      </section>
+    </section>
+
+    <section class="article-section last-section">
```
Comment (tirthajyoti-ghosh) [inline index.html:738]: There should be four boxes in each row in larger screens.

| Source Page	| Clone Page |
|:---:|:---:|
|image|image|

File: index.html (hunk 3 above)
Comment (tirthajyoti-ghosh) [inline index.html:251]: This section is overlapping with `latest news` section between screen width of `1024px` to `1301px`. Please fix this 👇 .

image

File: index.html (hunk 5 above)
Comment (tirthajyoti-ghosh) [inline index.html:738]: This section is encroaching on the footer.

image

File: index.html (hunk 4 above)
Comment (tirthajyoti-ghosh) [inline index.html:339]: Each box is encroaching on the other. Please fix this 👇 .

image

Comment (tirthajyoti-ghosh) [inline index.html:339]: In the mobile view, each box should take the full width of the screen.

| Source Page	| Clone Page |
|:---:|:---:|
|image|image|

File: index.html [hunk 6]
```diff
@@ -0,0 +1,806 @@
...
+  </main>
+  <footer>
+    <section class="footer-section-1">
+      <div class="dflex footer-section-1-div1 footer-links">
+        <div class="social-icons"> <i class="fab fa-facebook"></i></div>
+        <div class="social-icons"> <i class="fab fa-twitter"></i> </div>
+        <div class="social-icons"> <i class="fab fa-youtube"></i> </div>
+        <div class="social-icons"> <i class="fab fa-instagram-square"></i> </div>
+        <div class="social-icons"> <i class="fab fa-pinterest-p"></i> </div>
+        <div class="social-icons"> <i class="fas fa-envelope"></i> </div>
```
Comment (tirthajyoti-ghosh) [inline index.html]: The social icons should be links. Wrap them with `a` tags.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## What a remarkable project you have built 💯! This is impressive 🤩👌🧨!

### But in order to get it approved, we need to work on some changes. Please check the comments below.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

*Need help? Shoot me a message on Slack - [@Ghosh](https://app.slack.com/client/T47CT8XPG/DRDM6LAQN/user_profile/URB1MHXK6)*
<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,841 @@
...
+          </div>
+          <div class="funding-section-article-card-footer2">
+            <small>Mercuryo.io closes €2.5M Seed Funding</small>
+            <br>
+            <small class="font-small-3x">on FinSMEs</small>
+          </div>
+        </div>
+      </section>
+    </section>
+    <section class='dgrid2 article-section-short'>
```
Comment (tirthajyoti-ghosh) [inline index.html:339]: Let's decrease the space here 👇 .
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,841 @@
...
+        </div>
+      </div>
+      <nav class='second-nav-mobile xxl'>
+        <p>
+          Join TNW2020, our 15th annual flagship event. Online . Get tickets ➔
+        </p>
+      </nav>
+    </div>
+    <section class="first-article">
+      <div class='first-article-cards-small article-card-ipad-flex'>
```
Comment (tirthajyoti-ghosh) [inline index.html:118]: Let's reduce the space here 👇 .
image

File: index.html [hunk 9]
```diff
@@ -0,0 +1,841 @@
...
+  </main>
+  <footer>
+    <section class="footer-section-1">
+      <div class="dflex footer-section-1-div1 footer-links">
+        <div class="social-icons"> <i class="fab fa-facebook"></i></div>
+        <div class="social-icons"> <i class="fab fa-twitter"></i> </div>
+        <div class="social-icons"> <i class="fab fa-youtube"></i> </div>
+        <div class="social-icons"> <i class="fab fa-instagram-square"></i> </div>
+        <div class="social-icons"> <i class="fab fa-pinterest-p"></i> </div>
+        <div class="social-icons"> <i class="fas fa-envelope"></i> </div>
```
Comment (tirthajyoti-ghosh) [inline index.html]: The social icons should be links. Wrap them with `a` tags.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,841 @@
...
+          <button type="button">GET TICKETS</button>
+        </div>
+      </div>
+      <nav class='second-nav-mobile xxl'>
+        <p>
+          Join TNW2020, our 15th annual flagship event. Online . Get tickets ➔
+        </p>
+      </nav>
+    </div>
+    <section class="first-article">
```
Comment (tirthajyoti-ghosh) [inline index.html:117]: There is a bug in this section when the screen-width is around `1100px`.

image

The bug goes away below screen-height of `565px`. Please investigate and fix this.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Impressive progress so far 💪! Thanks for making all the changes 👏.

### However, we need to work on some changes before we can merge. Please check the comments below.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

*Need help? Shoot me a message on Slack - [@Ghosh](https://app.slack.com/client/T47CT8XPG/DRDM6LAQN/user_profile/URB1MHXK6)*
<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

File: index.html (hunk 9 above)
Comment (MuriloRoque) [reply index.html]: - [ ] Please, these icons should be links, make sure to wrap them with `a` tags.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @GzimAsani and @Ceejayski :wave: 

### Thank you for making almost all of the required changes, your project is looking good :clap: 

### But I believe you forgot to put your `a` tags in your social icons in the footer, please check my comment in your code below.

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello again @GzimAsani and @Ceejayski :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**